CloverSDK.podspec

```diff
@@ -0,0 +1,55 @@
+Pod::Spec.new do |s|
+  s.name         = "CloverSDK"
+  s.version      = "0.3.0"
+  s.summary      = "Clover SDK"
+  s.homepage     = "https://www.clover.com/features/integrations"
+  s.license      = { :type => "MIT", :file => "LICENSE" }
```

Comment:
The android sdk uses: Apache License, Version 2.0

I think it makes sense to use the same here.